roles/custom/matrix-appservice-draupnir-for-all/templates/production-appservice.yaml.j2

```diff
@@ -23,3 +23,9 @@ adminRoom: "{{ matrix_appservice_draupnir_for_all_master_control_room_alias }}"
 # This is a web api that the widget connects to in order to interact with the appservice.
 webAPI:
   port: 9000
+
+# The directory the bot should store various bits of information in
+dataPath: "/data"
+
+roomStateBackingStore:
+  enabled: {{ matrix_appservice_draupnir_for_all_enable_room_state_backing_store }}
```

Comment:
Using `to_json` is safer and more consistent with how we do things.

```diff
-  enabled: {{ matrix_appservice_draupnir_for_all_enable_room_state_backing_store }}
+  enabled: {{ matrix_appservice_draupnir_for_all_enable_room_state_backing_store | to_json }}
```

Of course, also see my other comment about renaming this variable. The code suggestion above is

Reply:
`to_json` is now used. The rest of this is addressed separately ofc but is hopefully addressed.